recipe/patches/0010-Empty-install-requirements-installed-by-conda-build.patch

```diff
@@ -1,59 +1,27 @@
-From 3c5886d8d47152735536fc7809908bbbad8041dd Mon Sep 17 00:00:00 2001
-From: Vasily Litvinov <vasilij.n.litvinov@intel.com>
-Date: Fri, 4 Dec 2020 12:49:34 +0300
-Subject: [PATCH 10/13] Empty install requirements - installed by conda build
+From 64a04a724d10186473776d2be86a72b26e752a0e Mon Sep 17 00:00:00 2001
+From: Kai Fricke <kai@anyscale.com>
+Date: Wed, 4 Aug 2021 11:40:39 +0100
+Subject: [PATCH 10/12] Empty install requirements - installed by conda build
 
 ---
- python/setup.py | 29 ++---------------------------
- 1 file changed, 2 insertions(+), 27 deletions(-)
+ python/setup.py | 4 ++--
+ 1 file changed, 2 insertions(+), 2 deletions(-)
 
 diff --git a/python/setup.py b/python/setup.py
-index f29fd74df..3c42da44c 100644
+index df87c1d42..dac4d9931 100644
 --- a/python/setup.py
 +++ b/python/setup.py
-@@ -118,32 +118,7 @@ extras["all"] = list(set(chain.from_iterable(extras.values())))
- # These are the main dependencies for users of ray. This list
- # should be carefully curated. If you change it, please reflect
- # the change in the matching section of requirements/requirements.txt
--install_requires = [
--    # TODO(alex) Pin the version once this PR is
--    # included in the stable release.
--    # https://github.com/aio-libs/aiohttp/pull/4556#issuecomment-679228562
--    "aiohttp",
--    "aiohttp_cors",
--    "aioredis",
--    "click >= 7.0",
--    "colorama",
--    "dataclasses; python_version < '3.7'",
--    "filelock",
--    "gpustat",
--    "grpcio >= 1.28.1",
--    "jsonschema",
--    "msgpack >= 1.0.0, < 2.0.0",
--    "numpy >= 1.16",
--    "protobuf >= 3.15.3",
--    "py-spy >= 0.2.0",
--    "pydantic >= 1.8",
--    "pyyaml",
--    "requests",
--    "redis >= 3.5.0",
--    "opencensus",
--    "prometheus_client >= 0.7.1",
--]
```

Comment:
I would rather you _not_ drop this part of the patch. It's here _specifically_ to cause a build failure by causing patch conflict if dependencies change, so the change doesn't go unnoticed.